### PR TITLE
Utilize pathlib in test_switch.py

### PR DESCRIPTION
**Pull request by berland**
(drive-by improvement)

### CODE REVIEW

File: tests/test_switch.py [hunk 1]
```diff
@@ -26,16 +25,16 @@ def test_write_activator_switches(tmpdir):
 else
     echo -e "{MIGRATION_WARNING}"
 fi
-"""
+""".strip()
+    )
 
-        assert actual.read() == expected
+    # assert actual.read() == expected
```
Comment (kwinkunks) [inline tests/test_switch.py:31]: Did you mean to leave this here?

Comment (berland) [reply tests/test_switch.py:31]: no, thanks!

**Review by kwinkunks — APPROVED** [review]
Other than the possibly extraneous comment, this looks great. So much cleaner than os.path, especially as tmpdir arrives as a Path. 👍

File: tests/test_switch.py (hunk 1 above)
Comment (berland) [reply tests/test_switch.py]: fixed

**Pull request merged**